### ISSUE TITLE
AutoHideTypes setting confusing

### ISSUE THREAD

**Issue by chevalierc** [xndcn/symbols-tree-view#131]
How are we suppose to inpput symbols???
```
variable, class, object, array, number, boolean
```
doessnt seem to work

**Comment by TenaciousDEK** [comment]
Me too. Auto-Hide-Types doesn't appear to work!
I would like to just be able to view the functions only, without listing all the variables. Is there any way to do that?

**Comment by nickschwab** [comment]
@chevalierc @TenaciousDEK It seems to require a restart of Atom before taking effect

**Comment by ilazarte** [comment]
No comma separation with single space and restart of atom worked for me.

**Comment by chevalierc** [comment]
@TenaciousDEK Got what we wanted based on @ilazarte 's comment. 

AutoHideTypes:
```
variable boolean unknown number object property undefined array null
```

**Comment by sergu44o** [comment]
hmmm. doesnt work for me. i tryed all variants + downgrade version. Doesnt work.

**Comment by vvinhas** [comment]
Indeed, It's very confusing. Maybe if we could select the available options would be better.

**Comment by oknixus** [comment]
Yes, it must be restart.

**Comment by fortinbras** [comment]
Does anybody konw how to hide python imports. For exemple import unicodedata

**Comment by opgobee** [comment]
In autohide field enter (copied from chevalierC):
variable boolean unknown number object property undefined array null
Do not put quotation marks around.
Restart atom
That worked for me
Only the unwanted symbol {c} still remains in the symbols-tree-view list, unclear where it points to in code, not constants.

**Comment by opgobee** [comment]
{c} seems to be comment. 
But adding 
comment
to the list of autohidetypes does not get rid of them

**Comment by opgobee** [comment]
Clicking right on the listing gives a popup where you can choose to show variables, functions or classes. 
The {c} regarded class, I could get rid of those entries by disabling 'class' in the popup list (but in my code it pointed to some comments, not classes). Using this popupbox might be even an easier way than putting things one doesn't want to see it in autohidetypes. And it works without restart atom.
